src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -138,14 +138,75 @@ public static CanvasDocument LoadFromSource(string directory2, ErrorContainer er
             // Load template files, recreate References/templates.json
             LoadTemplateFiles(errors, app, Path.Combine(directory2, PackagesDir), out var templateDefaults);
 
-            foreach (var file in dir.EnumerateFiles(AssetsDir))
-            {
+            // The resource entries for sample data is sharded into individual json files.
+            // Add each of these entries back into Resrouces.json
+            var resources = new List<ResourceJson>();
+            
+            foreach (var file in dir.EnumerateFiles(AssetsDir, "*", false))
+            {
+                var fileEntry = file.ToFileEntry();
+                // although should never be the case, but if it happens that both Resources.json file and individual sharded files exist, then read both.
                 if (file._relativeName == "Resources.json")
                 {
                     app._resourcesJson = file.ToObject<ResourcesJson>();
-                    continue;
                 }
```

Comment:
Are we actually backwards compatible here? If not, we should just remove this bit and bump the source version at the top of this file